src/ast/query.rs

```diff
@@ -64,6 +64,23 @@ impl fmt::Display for Query {
     }
 }
 
+/// Options for SetOperator
```

Comment:
Would be nice having more description about the structure. Maybe a reference of which big dialects use it...

Reply:
Okay, I added some comments at [ae6fb81](https://github.com/sqlparser-rs/sqlparser-rs/pull/689/commits/ae6fb81c13e74ae8240016c0203a146462b9ba42) .
But, I wasn't quite sure how to describe this enum...